src/libserver/cfg_rcl.c

```diff
@@ -2053,12 +2053,6 @@ rspamd_rcl_config_init (struct rspamd_config *cfg, GHashTable *skip_sections)
 				G_STRUCT_OFFSET (struct rspamd_config, history_rows),
 				RSPAMD_CL_FLAG_UINT,
 				"Number of records in the history file");
-		rspamd_rcl_add_default_handler (sub,
-				"disable_hyperscan",
-				rspamd_rcl_parse_struct_boolean,
-				G_STRUCT_OFFSET (struct rspamd_config, disable_hyperscan),
-				0,
-				"Disable hyperscan optimizations for regular expressions");
```

Comment:
This should not be removed - it is an used feature to disable HS from the configuration.